src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -11,8 +11,8 @@
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers should only contain numbers, and it should be between 3 to 20 digits long";
```

Comment:
Would `be 3 to 20 digits` be clearer since between brings up the debate on inclusive-of/exclusive-of?

Reply:
will do